alphagov/paas-cf
# [#157190168] Cleanup manifests following cf-deployment migration (part 2)

What
----

This is the second part of cleaning up the manifests following the cf-deployment migration. Part 1 (#1350) covered all the changes that would still leave a no-op deploy. The changes in this PR will cause a redeploy of all the VMs etc.

This PR covers a bunch of cleanup to bring our manifests closer to the cf-deployment manifests. See the individual commits for details of the changes.

How to review
-------------

Code review is probably easiest commit-by-commit.

* Deploy this branch on top of a deploy from master
* Check everything looks good.
* Rotate the leaf certificates (step 3 [here](https://team-manual.cloud.service.gov.uk/team/rotating_credentials/#rotating-certificates)), but **not the CA certs** (rotating CA certs is currently know to cause downtime)
* Redeploy from this branch with the rotated certs
* Verify all is good.

Who can review
--------------

Not @bandesz or myself.
